### ISSUE TITLE
Total issues, PRs and contributions seem wrong :(

### ISSUE THREAD

**Issue by chrisK824** [anuraghazra/github-readme-stats#2589]
### Describe the bug

Hi there and congrats for this cool tool. I've come aboard a few days now. I think the number of total issues, PRs and contributions seem wrong, at least in my case. I've definitely have more PRs and contributions, and even issues and that only with a quick look in a repo I'm contributing to. I've deployed a fork of the repo in my own vercel instance and have all private repos options enabled, still it seems that this isn't readint most of them.. :( 
![Screenshot_20230325_195825_Chrome](https://user-images.githubusercontent.com/79946379/227733930-52c9cf9a-3841-4c78-9533-e6b3c88e809f.jpg)

**Comment by chrisK824** [comment]
Hey @rickstaa , this is what I get from the explorer:

Query:

`
query MyQuery {
  viewer {
    issues {
      totalCount
    }
    pullRequests {
      totalCount
    }
    repositoriesContributedTo{
      totalCount
    }
    contributionsCollection {
      totalCommitContributions
      restrictedContributionsCount
      totalIssueContributions
      totalPullRequestContributions
    }
  }
}
`

Result:

`
{
  "data": {
    "viewer": {
      "issues": {
        "totalCount": 18
      },
      "pullRequests": {
        "totalCount": 22
      },
      "repositoriesContributedTo": {
        "totalCount": 2
      },
      "contributionsCollection": {
        "totalCommitContributions": 465,
        "restrictedContributionsCount": 178,
        "totalIssueContributions": 18,
        "totalPullRequestContributions": 22
      }
    }
  }
}
`

**Comment by chrisK824** [comment]
Not sure what you are referring to @rickstaa , I have deployed my own vercel instance with a fork of the repo. Is there some other option to distinguish vercel instances like "private" as you mention?

**Comment by rickstaa** [comment]
> Not sure what you are referring to @rickstaa , I have deployed my own vercel instance with a fork of the repo. Is there some other option to distinguish vercel instances like "private" as you mention?
> 
> I'm calling the API from my instance like this:
> 
> `https://github-readme-stats-chrisk824.vercel.app/api?username=chrisK824&show_icons=true&count_private=true&include_all_commits=true&theme=dark&card_width=500`

No, the thing I was referring to was that the PAT from [step 10-11](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) of the guide should have the right permissions (i.e. private repository read permissions).

![image](https://user-images.githubusercontent.com/17570430/228475813-2a04d703-bdee-4d16-aec0-30ac414d1944.png)

**Comment by chrisK824** [comment]
Hey @rickstaa , thanks for the feedback. I tried your suggestion and created a fine graind (beta0 personal token, however the implementation didn't work with that and I just received the error for not identified user.

However, putting back a new classic personal access token, did modify the data that came in. I'm pretty sure I had given all of the available permissions in options this time and last one as well, so it's a bit weird. In fact the data are now increased compared to previous but not sure if accurate. In any case, it's better than nothing 🤷🏽 

Thanks again. Cool project.
